### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by trav3711**
Project import generated by Copybara.

### CODE REVIEW

File: tools/abel/README.md [hunk 1]
```diff
@@ -69,33 +68,45 @@ dependencies are met:
...
+
+3. Create a token request file, `token_request.json` with contents:
 
     ```
-    {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+    {"scope": ["https://www.googleapis.com/auth/spreadsheets"],  "lifetime": "3600s"}
     ```
 
-3. Run the below curl command to obtain a token modifying the command with your
+4. Run the below curl command to obtain a token modifying the command with your
```
Comment (charbull) [inline tools/abel/README.md:97]: this sentence is a bit weird? rephrase?

File: tools/abel/README.md [hunk 2]
```diff
@@ -123,6 +134,12 @@ file. Only required for the `Building Config -> Spreadsheet` workflow.
   may be provided. Only required if the token is stored in a directory that isn't
   the current directory.
 
+`-p` or `--subscription` fully-qualified path to a Google Cloud Pubsub subscription, e.g. `projects/google.com:your-project/subscriptions/your-subscription`. This parameter is optional and it only required to run telemetry validation in a newly generated building config.
```
Comment (charbull) [inline tools/abel/README.md:137]: this could be removed?

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/abel/README.md (hunk 2 above)
Comment (trav3711) [reply tools/abel/README.md:137]: This is needed for telemetry validation if the user isn't using application-default authentication, no?

https://github.com/google/digitalbuildings/tree/master/tools/validators/instance_validator#telemetry-validation

File: tools/abel/README.md (hunk 1 above)
Comment (trav3711) [reply tools/abel/README.md:97]: Done.

**Review by tasodorff — APPROVED** [review]

File: tools/abel/README.md (hunk 2 above)
Comment (charbull) [reply tools/abel/README.md:137]: ok can you remove `is optional and it is only required` to this parameter is only required for telemetry validation.
it does not need to be newly generated right?

Comment (trav3711) [reply tools/abel/README.md]: Done.

File: tools/abel/README.md (hunk 1 above)
Comment (charbull) [reply tools/abel/README.md]: ` Run the below curl command to obtain a token. You need to modify the command with your`

File: tools/abel/README.md (hunk 2 above)
Comment (charbull) [reply tools/abel/README.md]: I dont see it updated, did you push it?

**Review by charbull — CHANGES_REQUESTED** [review]

**Comment by trav3711** [comment]
It looks like you're commenting on a version that's two commits behind.

File: tools/abel/README.md [hunk 3]
```diff
@@ -69,33 +68,45 @@ dependencies are met:
...
 
     ```
-    {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+    {"scope": ["https://www.googleapis.com/auth/spreadsheets"],  "lifetime": "3600s"}
     ```
 
-3. Run the below curl command to obtain a token modifying the command with your
-   service account from step #1. This command uses a dummy service account,
-   `your-service-account@project-id.iam.gserviceaccount.com`
+4. Run the following curl command to obtain a token by replacing dummy service account,
```
Comment (charbull) [inline tools/abel/README.md:97]: change those?

**Review by charbull — CHANGES_REQUESTED** [review]

**Comment by trav3711** [comment]
Added in the windows batch file and instructions for running the batch file.

**Pull request merged**